### DISCUSSION TITLE
冬天也吃得到火龍果？屏東智慧農業發威，用AI拉長產期、再增3億產值｜城市大小聲

### DISCUSSION THREAD

**Discussion by lukiea** [1101-GS3073-C/Python-Community-GS3073-C#724] [General]
https://futurecity.cw.com.tw/article/1517
將台灣這個水果帝國更加偉大

**Comment by Valentinaa0214** [comment]
雖然智慧農場的想法讓農民解決了火龍果滯銷的問題，但不知道會不會因為引入這種高科技設備，而使水果的價格飆升，感覺還有很多挑戰需要克服！

**Comment by yjpxi** [comment]
我覺得即使使用了高科技手段，極端氣候的變化仍然可能影響火龍果的生長，智慧農業雖然可以優化生產條件，但無法完全消除自然環境帶來的風險。
